src/iab.ts

```diff
@@ -125,7 +125,42 @@ export const TcfGvlV3DataCategory = t.type({
 export type TcfGvlV3DataCategory = t.TypeOf<typeof TcfGvlV3DataCategory>;
 
 /**
- * TCF GVL v3 stack configuration
+ * TCF GVL v3 URL configuration
+ */
+export const TcfGvlV3Urls = t.array(
+  t.intersection([
+    t.type({ langId: t.string, privacy: t.string }),
+    t.partial({
+      legIntClaim: t.string,
+    }),
+  ]),
+);
```

Comment:
I find 
```suggestion
export const TcfGvlV3Url = t.intersection([
  t.type({ langId: t.string, privacy: t.string }),
  t.partial({
    legIntClaim: t.string,
  }),
]);
```
a little more flexible. It's easier to wrap a type in `t.array` than to unravel it if you need.